public/locales/fr/common.json

```
@@ -1,47 +1,47 @@
 {
   "scrobble": "Scrobble",
   "getScrobbling": "Début du scrobbling",
-  "loading": "Chargement...",
+  "loading": "Chargement ...",
   "artist": "Artiste",
   "song": "Chanson",
   "title": "Titre",
   "album": "Album",
-  "scrobbleFromAlbum": "Scrobble albums",
-  "scrobbleAlbum": "Scrobble album",
+  "scrobbleFromAlbum": "Scrobbler des albums",
+  "scrobbleAlbum": "Scrobbler l'album",
   "album_plural": "Albums",
-  "goBack": "Go back",
-  "topAlbumsBy": "Albums by {{nameOfArtist}}",
-  "noAlbumsFound": "No albums found for <1>{{albumOrArtist}}</1>",
-  "noArtistsFound": "No artists found for <1>{{albumOrArtist}}</1>",
-  "emptyAlbum": "This album appears to be empty.",
-  "scrobbleSongs": "Scrobble songs",
-  "scrobbleSelected": "Scrobble selected",
-  "findAlbumCopy": "Enter an album or artist name",
-  "customTimestamp": "Custom timestamp",
-  "albumTimestampLogicDescription": "Scrobbling tracks \"now\" will scrobble the last track at the current time, backdating the previous ones accordingly (as if you had just finished listening). The \"custom timestamp\" will define the timestamp of the first track and add time to the following tracks from there (i.e. you specify the time you started listening to this album)",
+  "goBack": "Retour en arrière",
+  "topAlbumsBy": "Albums par {{nameOfArtist}}",
+  "noAlbumsFound": "Pas d'album trouvé pour <1>{{albumOrArtist}}</1>",
+  "noArtistsFound": "Pas d'artiste trouvé pour <1>{{albumOrArtist}}</1>",
+  "emptyAlbum": "Cet album semble être vide.",
+  "scrobbleSongs": "Scrobbler des chansons",
+  "scrobbleSelected": "Scrobble sélectionné",
```

In this case it can be one or many selected, do we leave it as it is or would you rather have `sélectionnés` instead? I should add a plurar form option in the future :grimacing:

---

This translation would be for the singular. If we want to put it in the plural, the translation should be: `Scrobbles sélectionnés`.
It's hard to say, but for a question of simplicity. I think the plural is better.

---

ok, we'll with that one then!

```suggestion
  "scrobbleSelected": "Scrobble sélectionnés",
```